pkg/release/release.go

```diff
@@ -23,7 +25,8 @@ type Config struct {
 	HelmClient *helmclient.Client
 	Logger     micrologger.Logger
 
-	Namespace string
+	Namespace  string
+	RestConfig *rest.Config
```

Comment:
I am not sure if we want to pass the rest config in here, or pass the individual clients. How do we want to handle this? Now both `host` and this gets a rest config and creates its own clients.

Reply:
I would prefer just creating the clients in one place and passing them around tbh. Uncertain.

Reply:
I was just worried about additional changes when we add more clients.
But maybe we should add all at once?

Reply:
You are right, let's pass the restconfig for now and do more changes later

Reply:
All the clients added.

Reply:
Ack, should be fine. We'll see.